Ira/movies-app/src/components/homePage.tsx

```diff
@@ -0,0 +1,78 @@
+import React from 'react';
+import Link from "react-router-dom";
+import Header from './header';
+import moviesData from './moviesData';
+import MovieCard from './movieCard'; 
+import NotFound from './notFound'; 
+import Layout from './layout';
+
+
+
+interface IHomePageState {
+   filteredMovies: Array<{}>;
+   searchInputValue: string;
+   searchBy: string;   
+   moviesCount: number;
+}
+
+class HomePage extends React.Component<IHomePageState> {   
+   
+   state = {
+      filteredMovies: [{}],   
+      searchInputValue: "",
+      searchBy: "title",
```

Comment:
Move "title", "genre" to ENUM. Strings that can be used several times in file better to move to CONST or ENUM (if you can group them, like: 
enum SortBy {
   Title: 'title',
    ...
})